src/llama_stack_client/resources/inference.py

```diff
@@ -213,7 +213,8 @@ def chat_completion(
         extra_body: Body | None = None,
         timeout: float | httpx.Timeout | None | NotGiven = NOT_GIVEN,
     ) -> InferenceChatCompletionResponse | Stream[InferenceChatCompletionResponse]:
-        extra_headers = {"Accept": "text/event-stream", **(extra_headers or {})}
+        if stream is True:
```

Comment:
this should be `if stream`, but the higher level issue is that this is generated code. We need to make sure we auto-apply this patch always after generation (see stainless_sync.sh) or find another way